### DISCUSSION TITLE
Correct way to set website url

### DISCUSSION THREAD

**Discussion by alonsoac** [LiveHelperChat/livehelperchat#1922] [General]
Hi. I am looking for the right setting so that email templates for reports include the correct url to the report.  With the default configuration the email is missing the folder name. For example say LHC is at mypage.com/lhc_web/index.php
The email link is: mypage.com//site_admin/user/login......   which is missing the folder lhc_web

I tried setting like this:
return array (
  'settings' => 
  array (
    'site' => 
    array (
      'site_address'=>'https://mypage.com/lhc_web/',

But this breaks for example the visitor widget because it now generates urls like mypage.com/lhc_web/lhc_web/

**Comment by remdex** [comment]
Can you reproduce on official demo as to hard to understand what you are clicking and where?

**Reply by alonsoac** [reply]
This cannot be reproduced in the demo because it only fails if the software is installed in a subfolder like https://mypage.com/lhc_web/
The problem is with the "Report prepared" email template. Specifically the {url_report} variable. 
I think in most parts of the software the url is determined from web server uri. But this template appears to be sent from cron so maybe it needs some setting to know in which folder the software is installed.

The link in the email is: mypage.com//site_admin/user/login...... which is missing the folder lhc_web

**Reply by remdex** [reply]
The thing is missing in Your bug report about what widget url you are talking?

**Reply by alonsoac** [reply]
No widget. Just the email for saved report. The url is in email is wrong and the report cannot be opened.  This is with no 'site_address' setting. If the the site_address is set like 'https://mypage.com/lhc_web/' then the url is ok in the email but the widget does not work because it duplicates the folder on urls like
 https://mypage.com//**lhc_web/lhc_web**/design/defaulttheme/fonts/MaterialIcons-lhc-v5.woff2
So at the moment I removed the setting and all is fine except the url in the email which is missing the folder lhc_web.

**Reply by remdex** [reply]
Where us that part?


'''
But this breaks for example the visitor widget because it now generates urls like mypage.com/lhc_web/lhc_web/
'''

**Reply by alonsoac** [reply]
Please disregar my comment about the widget as this seems to have caused some confusion. This is only about the link to the report is not gereated correctly.
I have found today that the link to password reminder is sent correctly.

Link on report email:  https://1.xxxx.online//site_admin/user/login......

Link on reminder email https://1.xxxx.online//lhc_web/index.php/site_admin/user/remindpassword....

**Reply by remdex** [reply]
So setup your server without lhc_web and index.php in path and all will be working
